paper/paper.md

```diff
@@ -161,40 +161,54 @@ def hybrid_function(x):
 	return jnp.sin(circuit(x)) ** 2
 ```
 
+```pycon
+>>> hybrid_function(0.543)
+array(0.70807342)
+```
+
 We can also consider an example that includes a classical optimization loop, such as optimizing a
 quantum computer to find the ground state energy of a molecule:
 
 ```python
+import pennylane as qml
+from catalyst import grad, for_loop, qjit
 import jaxopt
-from catalyst import grad
+from jax import numpy as jnp
 
-molecule = qml.data.load("qchem", molname="H3+")[0]
-n_qubits = len(molecule.hamiltonian.wires)
+mol = qml.data.load("qchem", molname="H3+")[0]
+n_qubits = len(mol.hamiltonian.wires)
 
 dev = qml.device("lightning.qubit", wires=n_qubits)
 
 @qjit
 @qml.qnode(dev)
 def cost(params):
-    qml.BasisState(molecule.hf_state, wires=range(n_qubits))
+    qml.BasisState(jnp.array(mol.hf_state), wires=range(n_qubits))
     qml.DoubleExcitation(params[0], wires=[0, 1, 2, 3])
     qml.DoubleExcitation(params[1], wires=[0, 1, 4, 5])
-    return qml.expval(molecule.hamiltonian)
+    return qml.expval(mol.hamiltonian)
 
 @qjit
-def workflow(init_params):
-    loss = lambda x: (cost(params), grad(cost)(params)[0])
```

Comment:
noticed a bug in the original code (note `x` vs. `params`)! this explains why optimization was happening so quickly.

Reply:
For me either version of this line runs about the same, ~280ms 🤔 

Any other changes to the file that could be relevant?

Or maybe it is indeed a regression in Catalyst.

Reply:
No other changes would be relevant 🤔 Maybe it's worth re-running this with Catalyst 0.3 or 0.4?